# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clucene/clucene-warnings.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- clucene-core-2.3.3.4/src/core/CLucene/analysis/AnalysisHeader.h	2012-02-22 12:37:22.531637934 +0000
+++ clucene-core-2.3.3.4/src/core/CLucene/analysis/AnalysisHeader.h	2012-02-22 12:39:15.369916728 +0000
@@ -7,6 +7,8 @@
 #ifndef _lucene_analysis_AnalysisHeader_
 #define _lucene_analysis_AnalysisHeader_

+#pragma GCC system_header
+
 #include "CLucene/index/Payload.h"
 #include "CLucene/util/VoidList.h"
 #include "CLucene/LuceneThreads.h"
--- clucene-core-2.3.3.4/src/core/CLucene/search/Searchable.h	2012-02-22 12:37:22.513637729 +0000
+++ clucene-core-2.3.3.4/src/core/CLucene/search/Searchable.h	2012-02-22 12:38:51.073641550 +0000
@@ -7,6 +7,7 @@
 #ifndef _lucene_search_Searcher_
 #define _lucene_search_Searcher_

+#pragma GCC system_header

 //#include "CLucene/index/IndexReader.h"
 CL_CLASS_DEF(index,Term)
--- clucene-core-2.3.3.4/src/core/CLucene/store/IndexInput.h	2012-02-22 12:37:22.508637673 +0000
+++ clucene-core-2.3.3.4/src/core/CLucene/store/IndexInput.h	2012-02-22 12:39:00.465747935 +0000
@@ -7,6 +7,8 @@
 #ifndef _lucene_store_IndexInput_
 #define _lucene_store_IndexInput_

+#pragma GCC system_header
+
 #include "CLucene/LuceneThreads.h"
 #include "CLucene/util/Equators.h"

--- clucene-core-2.3.3.4/src/core/CLucene/util/Array.h	2012-02-22 12:37:22.510637696 +0000
+++ clucene-core-2.3.3.4/src/core/CLucene/util/Array.h	2012-02-22 12:38:33.714444884 +0000
@@ -7,6 +7,8 @@
 #ifndef _lucene_util_Array_
 #define _lucene_util_Array_

+#pragma GCC system_header
+
 #include <stdlib.h>
 #include <string.h>

--- clucene-core-2.3.3.4/src/core/CLucene/util/PriorityQueue.h	2012-02-22 12:37:22.510637696 +0000
+++ clucene-core-2.3.3.4/src/core/CLucene/util/PriorityQueue.h	2012-02-22 12:38:40.316519685 +0000
@@ -7,6 +7,8 @@
 #ifndef _lucene_util_PriorityQueue_
 #define _lucene_util_PriorityQueue_

+#pragma GCC system_header
+
 #include <stdlib.h>

 CL_NS_DEF(util)
